README.md

```diff
@@ -29,6 +29,12 @@ Also read about:
 * [Options](https://marked.js.org/#/USING_ADVANCED.md)
 * [Extensibility](https://marked.js.org/#/USING_PRO.md)
 
+## Compatibility
+
+**Node.js:** Only [current and LTS](https://nodejs.org/en/about/releases/) Node.js versions are supported. End of life Node.js versions may become incompatible with Marked at any point in time.
+
+**Browser:** Not IE11 :)
```

Comment:
@UziTech Makes sense to also document the supported browsers but I can't seem to find them after a bit of light digging. Rollup doesn't seem to be using any specific browserlist for transpiling.

Reply:
it compiles to es5 but some functions (like `startsWith`) might need polyfills for browsers that don't include them.

Reply:
Let's just link to `package.json` so that can be the canonical place to determine which Node.js versions are supported.

We should also add this to the docs website, probably after the "Supported Markdown specifications" section.

Reply:
It might be worth to a set a rule which browsers are supported and to enforce it using https://www.npmjs.com/package/eslint-plugin-compat